### ISSUE TITLE
请问下有中文预训练模型开放嘛

### ISSUE THREAD

**Issue by yinnxinn** [open-mmlab/mmocr#124]
中文文档类的预训练模型。没有的话请问有没有计划

**Comment by jeffreykuang** [comment]
> 中文文档类的预训练模型。没有的话请问有没有计划

I have the plan to release pretrained chinese recognition model. Maybe in May 2021 if possible.
What is your application scenarios?  Any special requirements?

**Comment by yinnxinn** [comment]
thaks for reply . In our application , we usually get document image which taken by smartphone or other sanner device . So the ocr is a must

**Comment by cuhk-hbsun** [comment]
fix

**Comment by liuyunfeng2016** [comment]
你好请问一下，当前训练的预训练的图片数据和标注文件可不可以提供，想用satrn精度高的模型预训练模型再训练一下中文的数据集
label_digits_train.txt
label_handwriting_train.txt
label_printed_chinese_english_digits_train.txt
label_signatures_train.txt

**Comment by liuyunfeng2016** [comment]
> fix
你好请问一下，当前训练的预训练的图片数据和标注文件可不可以提供，想用satrn精度高的模型预训练模型再训练一下中文的数据集
label_digits_train.txt
label_handwriting_train.txt
label_printed_chinese_english_digits_train.txt
label_signatures_train.txt

**Comment by gaotongxiao** [comment]
@liuyunfeng2016 You can download ChineseOCR and randomly divide it into training and test set. The labels we used to train SAR are coarse and we are planning to refine them before releasing it.

**Comment by liuyunfeng2016** [comment]
> 

@gaotongxiao  好吧，那我只能等你们再发布了，多谢啦，因为SATRN的精度在其他开发的数据集上会比SAR高，所以我想在SATRN的基础是上进行中文预训练。感谢您的回复~
